src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn(
+        { 'TodoInfo TodoInfo--completed': todo.completed },
+        { TodoInfo: !todo.completed },
```

Comment:
You dont need this condition here. If you remove this condition and let className TodoInfo by default it will be the same logic.
```suggestion
       className={cn('TodoInfo', {
        'TodoInfo--completed': todo.completed,
      })}
```